pkg/plugins/plugindashboards/service.go

```diff
@@ -0,0 +1,151 @@
+package plugindashboards
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/bus"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/plugins"
+	pluginmodels "github.com/grafana/grafana/pkg/plugins/models"
+	"github.com/grafana/grafana/pkg/registry"
+	"github.com/grafana/grafana/pkg/tsdb"
+)
+
+func init() {
+	registry.Register(&registry.Descriptor{
+		Name:     "PluginDashboardService",
+		Instance: &Service{},
+	})
+}
+
+type Service struct {
+	TSDBService   *tsdb.Service          `inject:""`
```

Comment:
Wander what would be a better name for this service, DataRequestService?

Reply:
Done.